Mage/src/main/java/mage/cards/Card.java

```diff
@@ -40,12 +41,21 @@
 import mage.constants.Zone;
 import mage.counters.Counter;
 import mage.counters.Counters;
+import mage.filter.FilterMana;
 import mage.game.Game;
 import mage.game.GameState;
 import mage.game.permanent.Permanent;
 
 public interface Card extends MageObject {
 
+
+    final String regexBlack = ".*\\x7b.{0,2}B.{0,2}\\x7d.*";
```

Comment:
sure I can add the keyword, but what benefit would it give since it's static by default and applying static does not make it more static, the compiler would only give an extra warning there.